stardog-userrole-operator/templates/clusterrole.yaml

```diff
@@ -0,0 +1,117 @@
+---
```

Comment:
The ClusterRole should only be created if it is enabled in the configuration.

Reply:
This cluster role is for Operator's CRDs. If it does not exist, the Operator should not be able to work.

Reply:
Someone might want to manage cluster roles outside of Helm. I think we should allow this by making the creation of it configurable.